src/OSPSuite.Core/Domain/Constants.cs

```diff
@@ -568,6 +568,7 @@ public static class ParameterExport
 
       public const string NOT = "Not";
       public const string AND = "and";
+      public const string OR = "or";
```

Comment:
why are "or" and "and" written in small letters and "Not" not? 😄

Reply:
good point. At the end of the day, we show them in ALL CAPS. In can update next time